lib/smart_app_launch/backend_services_group.rb

```diff
@@ -0,0 +1,237 @@
+require_relative 'authorization_request_builder'
+
+module SMARTAppLaunch
+  class SMARTBackendServices < Inferno::TestGroup
+    title 'Backend Services'
+    short_description 'Demonstrate SMART Backend Services Authorization'
+
+    id :smart_backend_services
+
+    input :bulk_token_endpoint,
+          title: 'Backend Services Token Endpoint',
+          description: <<~DESCRIPTION
+            The OAuth 2.0 Token Endpoint used by the Backend Services specification to provide bearer tokens.
+          DESCRIPTION
+    input :bulk_client_id,
+          title: 'Bulk Data Client ID',
```

Comment:
These titles should be updated as well.

Reply:
All titles/inputs/descriptions have been updated so they no longer refer to bulk data.